doc/source/user_guide/session/launching_ansys_fluent.rst

```diff
@@ -380,3 +380,76 @@ Connecting to a Fluent container running inside Linux from a Windows host
 
   >>> import ansys.fluent.core as pyfluent
   >>> solver = pyfluent.connect_to_fluent(ip="10.18.19.151", port=44383, password="hbsosnni")
+
+
+Connecting to Fluent on Windows from a Linux or WSL Host
+--------------------------------------------------------
+
+This guide describes how to connect to an ANSYS Fluent instance running on a Windows machine from a Linux or WSL host. It also includes steps to enable remote file transfer.
+
+A. **Set Up Fluent and File Transfer Server on Windows**
+
+1. **Launch Fluent**
+
+   Open a command prompt and run:
+
+   .. code:: console
+
+      ANSYS Inc\v252\fluent\ntbin\win64\fluent.exe 3ddp -sifile=server_info.txt
+      type server_info.txt
+
+   Example output:
+   ``10.18.44.179:51344``, ``5scj6c8l``
+
+2. **Retrieve Connection Details**
+
+   Get the IP address, port, and password from the `server_info.txt` file.  
+   Example:
+   - IP: `10.18.44.179`
+   - Port: `51344`
+   - Password: `5scj6c8l`
+
+3. **Set Working Directory**
+
+   Change to the Fluent installation directory:
+
+   .. code:: console
+
+      cd "ANSYS Inc\v252\fluent\ntbin\win64"
+
+4. **Start the File Transfer Server**
+
+   Launch the container:
+
+   .. code:: console
+
+      docker run -p 50000:50000 -v %cd%:/home/container/workdir ghcr.io/ansys/tools-filetransfer:latest
```

Comment:
Is this image available outside Ansys?

Reply:
The package is marked as internal (https://github.com/ansys/ansys-tools-filetransfer-server/pkgs/container/tools-filetransfer). I think it is within ansys org only.

We can point users to the [public documentation](https://filetransfer-server.tools.docs.pyansys.com/version/stable/intro.html#) on how to build the server. That public documentation doesn't also mention the ghcr image.

Reply:
Done. Thank you.